prow/cmd/webhook-server/main.go

```diff
@@ -224,6 +230,7 @@ func handleSecrets(client ClientInterface, ctx context.Context, clientoptions cl
 		}
 		cert = secretsMap[certFile]
 		privKey = secretsMap[privKeyFile]
+		caPem = secretsMap[caBundleFile]
```

Comment:
We have discussed early on that this is not the best way of representing a map with fixed keys, can we change this to:
```
type certBundle struct {
  Cert string
  PrivKey string
  CaPem string
}
```

To avoid function passing three unamed variables (e.g. `func () {string, string, string, error)`) around?